EAD-Assignment-5/src/com/metacube/advertismentwebservices/facade/AdvertisementFacade.java

```diff
@@ -0,0 +1,116 @@
+package com.metacube.advertismentwebservices.facade;
+
+import java.util.List;
+
+import com.metacube.advertismentwebservices.Common.Status;
+import com.metacube.advertismentwebservices.dao.AdvertisementDao;
+import com.metacube.advertismentwebservices.dao.CategoryDao;
+import com.metacube.advertismentwebservices.dao.AdvertisementDao;
+import com.metacube.advertismentwebservices.dao.CategoryDao;
+import com.metacube.advertismentwebservices.model.Advertisement;
+import com.metacube.advertismentwebservices.model.Category;
+
+/**
+ * Facade Class for advertisement table
+ */
+public class AdvertisementFacade {
+	private static AdvertisementFacade advertisementFacade = new AdvertisementFacade();
+
+	AdvertisementDao advertisementDao = (AdvertisementDao) AdvertisementDao
+			.getInstance();
+	CategoryDao categoryDao = (CategoryDao) CategoryDao.getInstance();
```

Comment:
I don't think we need to Type Cast it here. If any specific reason please reply on this comment.